bin/update_pythons.py

```diff
@@ -196,27 +196,32 @@ def __init__(self) -> None:
                 uri = int(release["resource_uri"].rstrip("/").split("/")[-1])
                 self.versions_dict[version] = uri
 
-    def update_version_macos(self, identifier: str, spec: Specifier) -> ConfigMacOS | None:
-        file_idents = ("macos11.pkg", "macosx10.9.pkg", "macosx10.6.pkg")
+    def update_version_macos(
+        self, identifier: str, version: Version, spec: Specifier
+    ) -> ConfigMacOS | None:
         sorted_versions = sorted(v for v in self.versions_dict if spec.contains(v))
 
-        for version in reversed(sorted_versions):
+        if version <= Version("3.8.9999"):
+            file_ident = "macosx10.9.pkg"
+        else:
+            file_ident = "macos11.pkg"
```

Comment:
We could instead take the old file identifier, and only look for that.

Reply:
This is fine for now, but that's a possible update later to remove the explicit version check.